native/markdown/rules.react.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import type { StyleSheetOf } from '../themes/colors';
+import type { MarkdownStyles } from './styles';
+
+import * as React from 'react';
+import { Text, Linking } from 'react-native';
+import * as SimpleMarkdown from 'simple-markdown';
+
+export default function rules(styles: StyleSheetOf<MarkdownStyles>) {
+  return {
+    // Matches '<https://google.com>' during parse phase and returns a 'link'
```

Comment:
I shortened most of your comments. Sorry if it feels like I wasted your time with that. I mostly just wanted it to be clear what each rule was matching and how the AST looked